code/datums/spell.dm

```diff
@@ -131,18 +127,36 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell))
 	var/special_availability_check = 0//Whether the spell needs to bypass the action button's IsAvailable()
 
 	var/sound = null //The sound the spell makes when it is cast
-	/// If the ability is for vampires
-	var/vampire_ability = FALSE
-	var/required_blood = 0
 	var/gain_desc = null
-	var/deduct_blood_on_cast = TRUE
+
+	/// The message displayed when a click based spell gets activated
+	var/selection_activated_message		= "<span class='notice'>Click on a target to cast the spell.</span>"
+	/// The message displayed when a click based spell gets deactivated
+	var/selection_deactivated_message	= "<span class='notice'>You choose to not cast this spell.</span>"
+
+	/// does this spell generate attack logs?
+	var/create_attack_logs = TRUE
+
+	/// If this spell creates custom logs using the write_custom_logs() proc. Will ignore create_attack_logs
+	var/create_custom_logs = FALSE
+
+	/// Which targeting system is used. Set this in create_new_targeting
+	var/datum/spell_targeting/targeting
+	/// List with the targeting datums per spell type. Key = src.type, value = the targeting datum created by create_new_targeting()
+	var/static/list/targeting_datums = list()
+
+	/// Which spell_handler is used in addition to the normal spells behaviour, can be null. Set this in create_new_handler if needed
+	var/datum/spell_handler/custom_handler
+	/// List with the handler datums per spell type. Key = src.type, value = the handler datum created by create_new_handler()
+	var/static/list/spell_handlers = list()
 
 /* Checks if the user can cast the spell
  * @param charge_check If the proc should do the cooldown check
  * @param start_recharge If the proc should set the cooldown
  * @param user The caster of the spell
 */
 /obj/effect/proc_holder/spell/proc/cast_check(charge_check = TRUE, start_recharge = TRUE, mob/user = usr) //checks if the spell can be cast based on its settings; skipcharge is used when an additional cast_check is called inside the spell
+	// SHOULD_NOT_OVERRIDE(TRUE) Todo for another refactor
```

Comment:
I couldn't dive into another rabbit hole anymore. This can be tackled later if needed. Only really needed to clean up the code. No functional changes